src/App.jsx

```diff
@@ -1,34 +1,24 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
 import "./App.css";
+import Header from "./components/header/Header";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticles } from "./utils/api";
 
 function App() {
-  const [count, setCount] = useState(0);
+  const [items, setItems] = useState([]);
```

Comment:
I would try to be a bit more descriptive with your state. The API call you're making is getting `articles` so I would name it that, rather than items 👍